Without this do_configure was trying to create /include/WebCore/libdummy.prl in root of build host filesystem
now it's in proper place ${WORKDIR}/qt-everywhere-opensource-src-4.8.1/src/3rdparty/webkit/include/WebCore/libdummy.prl

First reported here:
http://www.mail-archive.com/openembedded-devel@lists.openembedded.org/msg24436.html

Upstream-Status: Pending

Open-webOS-DCO-1.0-Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>

--- qt-everywhere-opensource-src-4.8.1/src/3rdparty/webkit/Source/WebKit2/DerivedSources.pro.orig	2012-05-01 07:48:08.457613310 +0200
+++ qt-everywhere-opensource-src-4.8.1/src/3rdparty/webkit/Source/WebKit2/DerivedSources.pro	2012-05-01 07:48:15.493613232 +0200
@@ -3,6 +3,8 @@
 
 CONFIG -= debug_and_release
 
+isEmpty(OUTPUT_DIR): OUTPUT_DIR = ../..
+
 CONFIG(standalone_package) {
     isEmpty(WEBKIT2_GENERATED_SOURCES_DIR):WEBKIT2_GENERATED_SOURCES_DIR = $$PWD/generated
     isEmpty(WC_GENERATED_SOURCES_DIR):WC_GENERATED_SOURCES_DIR = $$PWD/../WebCore/generated
